### ISSUE TITLE
Converting ONNX model to MNN reshapes it in a way I cannot use

### ISSUE THREAD

**Issue by ScottJohnson2718** [alibaba/MNN#1971]
# 平台(如果交叉编译请再附上交叉编译目标平台):  
# Platform(Include target platform as well if cross-compiling):  
Windows 10

# Github版本:
# Github Version: 1.2.7

直接下载ZIP包请提供下载日期以及压缩包注释里的git版本(可通过``7z l zip包路径``命令并在输出信息中搜索``Comment`` 获得，形如``Comment = bc80b11110cd440aacdabbf59658d630527a7f2b``)。 git clone请提供 ``git commit`` 第一行的commit id

# 编译方式:
# Compiling Method

```
请在这里粘贴cmake参数或使用的cmake脚本路径以及完整输出
Paste cmake arguments or path of the build script used here as well as the full log of the cmake proess here or pastebin
```
cmake.exe ^
	-G "Visual Studio 16 2019" ^
	-Ax64 ^
	-S..\MNN ^
	-B\work\mnn_vs2019_vulkan ^
	-DCMAKE_CONFIGURATION_TYPES="Debug;Release" ^
	-DMNN_VULKAN=ON ^
	-DMNN_BUILD_DEMO=ON ^
	-DMNN_BUILD_TOOLS=ON ^
	-DMNN_BUILD_CONVERTER=ON ^
	-DMNN_BUILD_SHARED_LIBS=ON ^
	-DMNN_AVX512=ON ^
	-DMNN_WIN_RUNTIME_MT=ON ^
	-DMNN_BUILD_PROTOBUFFER=ON ^
	-DMIN_BUILD_EVALUATION=ON  ^
	-DCMAKE_INSTALL_PREFIX="c:/work/mnn_vs2019_vulkan_install" 
	
# 编译日志:
# Build Log:
PS C:\git_repos\MNN> .\build_vs2019.bat
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.11.15
** Copyright (c) 2021 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
--
-- 3.19.0.0
-- Use Threadpool, forbid openmp
-- >>>>>>>>>>>>>
-- MNN BUILD INFO:
--      System: Windows
--      Processor: AMD64
--      Metal: OFF
--      OpenCL: OFF
--      OpenGL: OFF
--      Vulkan: ON
--      ARM82: OFF
--      oneDNN: OFF
--      TensorRT: OFF
--      CoreML: OFF
--      CUDA: OFF
--      OpenMP: OFF
--      BF16: OFF
--      ThreadPool: ON
--      Hidden: TRUE
--      Build Path: /work/mnn_vs2019_vulkan
-- WIN_USE_ASM:
-- AMD64: Open SSE
-- MNN_AVX512:ON
-- Onnx:
-- MNN Depends:MNN
-- Configuring done
-- Generating done
-- Build files have been written to: /work/mnn_vs2019_vulkan
PS C:\git_repos\MNN>
```
粘贴在这里
Paste log here or pastebin
```
Converted Success!
PS C:\work\mnn_vs2019_vulkan\Release> .\MNNConvert.exe -f ONNX --modelFile yolov5s.onnx --MNNModel yolov5s.mnn --bizCode MNN --optimizeLevel 0
Converting
Start to Convert Other Model Format To MNN Model...
[09:20:02] :30: ONNX Model ir version: 6
Start to Optimize the MNN Net...
inputTensors : [ images, ]
outputTensors: [ 397, 458, 519, output, ]
Converted Success!

My question is how to read the output of the MNN inference? I have the MNN-yolo3 code as an example. 
I am trying to use the Ultralytics yolov5s.onnx model in MNN. I have code in C# that reads in the output of the ONNX model correctly as an example to follow. This shows me the layout of the output tensor data and how to parse the bounding boxes and confidences out of the ONNX model for yolov5s.onnx.
 I convert the ONNX model to MNN using the command shown above and use sample code from MNN-Yolov3 project to read the output. The output shape of the MNN model is [1,3,80,80,85]. This is not the same shape as the ONNX model. So I tried converting the ONNX model to MNN with the option --optimizeLevel 0 and that has no effect on the output. I understand that the MNN-yolov3 code is just an example and I must change the code for the correct shape of the output tensor from yolov5s.mnn. 

How can I determine how to read the output data from the MNN converted model since it was changed from the ONNX model?

**Comment by jxt1234** [comment]
MNN will not change the output shape of ONNX. You can use fastTestOnnx.py to make testdir and then use --testdir to check it.

**Comment by ScottJohnson2718** [comment]
Thanks for the information. This allowed me to seek the problem elsewhere. The problem was that there were multiple outputs in the onnx model. It took a while to realize that the sample code for using the model requested the default output (by passing in null) and that was the [1,3,80,80,85] data. The actual output was named "output" and when I changed the application code to specifically use the node named "output", it returned the correct data and the application worked. I had to use netron to see the nodes and I stepped through the MNN converter to make sure the output node was converted to MNN.
